mandrill-api/mandrill-api.d.ts

```diff
@@ -0,0 +1,1182 @@
+// Type definitions for Mandrill API 1.x
+// Project: http://mandrill.com/

```

Comment:
it was pointing to madrill.com (was missing the n). yes, it's mandrill from mailchimp